src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1675,6 +1692,17 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
                 }
             }
 
+            if (PlatformDetection.IsUap)
```

Comment:
Nit: here and above: wouldn't it be more readable if it has a single `if (IsUap && method=="TRACE")` ?